debian/control

```diff
@@ -1,20 +1,22 @@
 Source: hildon-input-method
 Section: x11
 Priority: optional
-Maintainer: Mohammad Anwari <Mohammad.Anwari@nokia.com>
-Build-Depends: debhelper, cdbs, pkg-config, libgtk2.0-dev (>= 2.14.7-1maemo15), libosso-dev (>=0.9.10-2), libgconf2-dev, libhildon1-dev, hildon-input-method-framework-dev (>= 1:2.1.66-1), libdbus-1-dev, maemo-launcher-dev (>= 0.23-1), gtk-doc-tools
+Maintainer: Ivaylo Dimitrov <ivo.g.dimitrov.75@gmail.com>
```

Comment:
Changing maintainer is important thing - it's better to be done in a separate commit

Reply:
Sure, if I was posting on LKML, which I am not.

Reply:
Don't you take Maemo Leste seriously?

Reply:
are you kidding me?

Reply:
I think if the things will go well, more developers will join us. In such a case it's better to show how things should be done properly from start. Instead of talking about LKML > Maemo-Leste. Who cares?